prebuild/README.md

```diff
@@ -2,9 +2,16 @@
 
 ## Get Debian base boxes
 
+### For the 64 bit box
+
 ```bash
 vagrant box add debian/contrib-jessie64
 ```
+### For the 32 bit box
+
+```bash
+vagrant box add bento/debian-8.6-i386
```

Comment:
This is what is recommended by the Vagrant documentation: https://www.vagrantup.com/docs/boxes.html (see Official Boxes paragraph).